content/influxdb/cloud-dedicated/process-data/downsample/downsample-quix.md

```diff
@@ -32,7 +32,7 @@ This guide also assumes you have already
 
 The following diagram illustrates how data is passed between processes as it is downsampled:
 
-{{< html-diagram/quix-downsample-pipeline >}}
+{{< html-diagram/influxdb-v3-quix-downsample-pipeline >}}
 
 {{% note %}}
 It is usually more efficient to write raw data directly to Kafka rather than
```

Comment:
Line 102: you could simplify as JSON is default, so you don't need the serializers/deserializers.

![image](https://github.com/quixio/partner-docs-influxdb/assets/25511700/f1cfc2a8-9e29-435c-be2e-76f9d5ff7318)